Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1255,6 +1255,9 @@
 /* Label text in device manager for a device with no name */
 "UNNAMED_DEVICE" = "Unnamed Device";
 
+/* Named used for groups without a name. */
+"UNNAMED_GROUP_NAME" = "Unnamed Group";
```

Comment:
I'm going to revert this and keep using our existing default name "New Group" until we implement your idea of "Alice, Bob, etc."